### DISCUSSION TITLE
Trackpad gestures conflict with macOS accessibility settings

### DISCUSSION THREAD

**Discussion by F1248** [rxhanson/Charmstone-Community#33] [Ideas]
If "Use trackpad gesture to zoom" is enabled in the "Zoom" section of the accessibility settings, the trackpad gestures no longer work.
I know that this probably only affects very few users but for those like me it is very annoying when this very useful tool that you have already gotten used to suddenly stops working for seemingly no reason, it took me a very long time to find the cause.
Maybe it is possible to check this setting and then display a hint?

**Comment by rxhanson** [comment]
Sorry that you experienced this, but thanks for troubleshooting it and letting me know. This starts to get into the gray areas of gesture processing in macOS where some users might not experience this with the same exact settings, but I’ll definitely add in some messaging here, as you recommended.
